### PR TITLE
Multi service dev mode

### LINKED ISSUE #242: Command to start multiple okteto environments at the same time
**Is your feature request related to a problem? Please describe.**
Some developers need to start more than one service at a time when developing. Something similar to what 'docker-compose' does for docker would be very useful.

### PR DESCRIPTION
**Pull request by pchico83**
Closes #242

### CODE REVIEW

File: cmd/up.go [hunk 1]
```diff
@@ -290,26 +301,36 @@ func (up *UpContext) devMode(isRetry bool) error {
...
-	c, err := deployments.DevModeOn(d, up.Dev, create, up.Client)
-	if err != nil {
+	if err := deployments.TraslateDevMode(deploys, up.Dev); err != nil {
 		return err
 	}
 
-	up.Container = c.Name
+	for name, d := range deploys {
+		forceCreate := false
+		if d.Name == name {
```
Comment (rberrelleza) [inline cmd/up.go]: what is this check for?

Comment (pchico83) [reply cmd/up.go]: it is to handle de autocreation of the main service. for the other ones, for now, they must be already created

Comment (rberrelleza) [reply cmd/up.go]: could we have that separate from the loop so it's more obvious? Or maybe have a function called "isMainService" so it's easier to troubleshoot in the future?

File: pkg/k8s/deployments/translate.go [hunk 2]
```diff
@@ -238,20 +258,3 @@ func translateOktetoVolumes(d *appsv1.Deployment, dev *model.Dev) {
 		d.Spec.Template.Spec.Volumes = append(d.Spec.Template.Spec.Volumes, v)
 	}
 }
-
-func getPullPolicy() apiv1.PullPolicy {
```
Comment (rberrelleza) [inline pkg/k8s/deployments/translate.go]: Are we removing the support for this? I thought this was breaking quid's use ase

Comment (pchico83) [reply pkg/k8s/deployments/translate.go]: I talked to alberto the other day. he said we should keep the policy defined in the deployment manifest. It is what we are doing for other fields, I think we should follow that approach here too.

**Pull request merged**